packages/markdown-it-synapse/src/index.ts

```diff
@@ -272,16 +274,16 @@ module.exports = function synapse_plugin(md, _suffix, _baseURL) {
   md.inline.ruler.after('emphasis', 'synapse', synapse)
 }
 
-module.exports.footnotes = function () {
+function getFootnotes() {
```

Comment:
Renamed function locally to not clash with existing `footnotes` variable. Will be renamed back to `footnotes` on export, so this isn't a breaking change.